sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TablesExtensionConfigProvider.cs

```diff
@@ -227,5 +238,28 @@ private static TableEntity CreateTableEntityFromJObject(JObject entity)
 
             return tableEntity;
         }
+
+        // Cannot use `new BinaryData(tableAttribute)` because this "may only be called on a Type for
+        // which Type.IsGenericParameter is true"; therefore we use our own reference type.
```

Comment:
You get a runtime exception when you attempt this? I tried this my self and I don't see the error you mention, I get a serialization error *"System.NotSupportedException: 'Serialization and deserialization of 'System.RuntimeType' instances are not supported. The unsupported member type is located on type 'System.Object'. Path: $.TypeId.'"*.  Perhaps it's the case that attribute instances can never be serialized in this manner.

That's probably best since this is part of the contract for the binding, so a dedicated contract type is safest.

Reply:
Yeah I get that specific error when I do the E2E testing with the converter